src/components/Collapsible/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { ReactNode } from "react";
+
+interface CollapsibleProps {
+    title?: ReactNode;
+    children?: ReactNode;
+    isCollapsed?: boolean;
+}
+
+export function Collapsible(props: CollapsibleProps) {
+    return (
+        <div>
+            <div>{props.title}</div>
+            {!props.isCollapsed && <div className="pl-3">{props.children}</div>}
```

Comment:
`pl-3` - it is `padding-left: 3`?

Reply:
`pl-3` is for `padding-left: 0.75rem; /* 12px */`. Tailwind has a set of pre-defined sizes that fit pretty well in my opinion the majority of use-cases. Tailwind devs maintain that it's a good idea to have a finite set of choices, because it eases decision-making as well as consistency of spacings across the app.

https://tailwindcss.com/docs/padding